bin/node/rpc/src/lib.rs

```diff
@@ -115,6 +120,8 @@ pub fn create_full<C, P, SC>(
 	C::Api: BlockBuilder<Block>,
 	P: TransactionPool + 'static,
 	SC: SelectChain<Block> +'static,
+	B: Send + Sync + 'static + sc_client_api::Backend<Block>,
+	<B as sc_client_api::Backend<Block>>::State: sp_state_machine::Backend<BlakeTwo256>,
```

Comment:
```suggestion
	B: sc_client_api::Backend<Block> + Send + Sync + 'static,
	B::State: sp_state_machine::Backend<sp_runtime::traits::HashFor<Block>>,
```

Reply:
I think the reason we need to specify the state backend here is because `Block` isn't a generic type parameter and is actually the concrete block type. Either way doing it this way is more generic since we don't tie the bound to the hashing we currently use.